kong/plugins/aws-lambda/handler.lua

```diff
@@ -129,6 +129,7 @@ function AWSLambdaHandler:access(conf)
   AWSLambdaHandler.super.access(self)
 
   local upstream_body = new_tab(0, 6)
+  local var = ngx.var
```

Comment:
Mind removing the below `local var = ngx.var` line that now shadows this definition?